src/components/StorageOverview/Capacity/Capacity.js

```diff
@@ -1,38 +1,88 @@
+
 import { StorageOverviewContext } from '../StorageOverviewContext';
 import { CapacityItem } from '../../Dashboard/Capacity/CapacityItem';
 import { formatBytes } from '../../../utils';
 import { getCapacityStats } from '../../../selectors';
 import { CapacityBody } from '../../Dashboard/Capacity/CapacityBody';
+import { TOTAL, CAPACITY, VMSVSPODS } from './strings';
+
+const capacityTypes = [TOTAL, CAPACITY, VMSVSPODS];
 
 export class Capacity extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      capacity: capacityTypes[0],
+    };
+  }
+
+  convertMetrics = (total, used, title) => ({ total, used, title });
+
+  changeCapacityMetric = newVal => {
+    this.setState({ capacity: newVal }, this.getNewCapacityMetric);
+  };
+
+  getNewCapacityMetric = () => {
+    switch (this.state.capacity) {
+      case TOTAL:
+        return this.convertMetrics(this.props.capacityTotal, this.props.capacityUsed, TOTAL);
+      case CAPACITY:
+        return this.convertMetrics(this.props.capacityRequested, this.props.capacityUsed, CAPACITY);
+      case VMSVSPODS:
+        return this.convertMetrics(this.props.podsCapacity, this.props.vmsCapacity, VMSVSPODS);
+      default:
+        return [];
+    }
+  };
+
   render() {
-    const { capacityTotal, capacityUsed, LoadingComponent } = this.props;
+    const { capacityTotal, capacityRequested, capacityUsed, vmsCapacity, podsCapacity } = this.props;
+
+    const capacitymetric = this.getNewCapacityMetric();
+    const isLoading = !capacityTotal && !capacityRequested && !capacityUsed && !vmsCapacity && !podsCapacity;
+
     return (
       <DashboardCard className="kubevirt-capacity__card">
         <DashboardCardHeader>
-          <DashboardCardTitle>Capacity</DashboardCardTitle>
+          <Row>
+            <Col lg={6} md={6} sm={6} xs={6}>
+              <DashboardCardTitle>Capacity</DashboardCardTitle>
+            </Col>
+            <Col>
+              <DashboardCardActionsBody>
+                <Dropdown
+                  id="capacity-type"
+                  value={this.state.capacity}
+                  onChange={this.changeCapacityMetric}
+                  choices={capacityTypes}
+                  disabled={isLoading}
+                  groupClassName="kubevirt-dropdown__group"
+                />
+              </DashboardCardActionsBody>
+            </Col>
+          </Row>
         </DashboardCardHeader>
         <DashboardCardBody>
           <CapacityBody>
             <CapacityItem
               id="capacity"
-              title="Total capacity"
-              used={getCapacityStats(capacityUsed)}
-              total={getCapacityStats(capacityTotal)}
+              title={capacitymetric.title}
+              used={getCapacityStats(capacitymetric.used)}
+              total={getCapacityStats(capacitymetric.total)}
               formatValue={formatBytes}
-              LoadingComponent={LoadingComponent}
```

Comment:
why did you remove passing LoadingComponent ? We need it to pass `LoadingInline` from `web-ui`